createAccountCore/src/main/java/com/hamdan/forzenbook/createaccount/core/viewmodel/BaseCreateAccountViewModel.kt

```diff
@@ -30,7 +30,7 @@ abstract class BaseCreateAccountViewModel(
 
     protected abstract var createAccountState: CreateAccountState
 
-    lateinit var onAccountCreateSuccess: () -> Unit
+    open lateinit var onAccountCreateSuccess: () -> Unit
```

Comment:
Just make this a nullable lambda. I've never seen an open lateinit var in all my years lol

Reply:
gotcha